docs/migration/enact/migrating-to-enact-3.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Migrating to Enact 3.0
+---
+
+## Overview
+This document lists changes between Enact versions 2.x and 3.0 likely to affect most apps.  If you
+are coming from Enact 1.x, please [migrate to 2.0](./migrating-to-enact-2.md) and then consult
+this guide.
+
+## General Changes
+
+### webOS TV
```

Comment:
Can you add something here about how spotlight now allows focusing to disabled items?